src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -59,7 +59,7 @@ export const authOptions: AuthOptions = {
       authorization: {
         url: AppConstants.OAUTH_AUTHORIZATION_URI,
         params: {
-          scope: "profile",
+          scope: "profile https://identity.mozilla.com/account/subscriptions",
```

Comment:
Q: Should this be hardcoded URL, or should it be from an env var so we can toggle between stage/prod based on deployment?

Reply:
It's actually the name of the scope (similar to `profile`) which is admittedly somewhat confusing :) I think it's being used as a URI not URL in this case.

Reply:
https://mozilla.github.io/ecosystem-platform/api#tag/OAuth-Server-API-Overview/operation/postIntrospect for reference